### ISSUE TITLE
Installation step-by-step differs for Xcode 5

### ISSUE THREAD

**Issue by tuzz** [SVGKit/SVGKit#113]
I'm new to iOS programming and it's quite hard to follow the installation steps. I think this is due to Xcode updates and a few inconsistencies between the explanation and the user interface.

If possible, could you audit the installation steps and make sure they're still correct?

Thanks

**Comment by johanndiedrick** [comment]
I'm also having the same problems as well. I'm having a little difficulty following how to add SVGKit to an existing project with Xcode 5. Any clarification would be much appreciated!

**Comment by tuzz** [comment]
I used this: https://github.com/MaddTheSane/SVGKit

Still not absolutely clear, but much better.

Cheers

**Comment by adamgit** [comment]
Sorry about this ... I'm experimenting with different ways of doing this, to workaround the new bugs in Xcode5.

Some of Xcode5 makes this a _lot_ easier than it was in Xcode4, but some of the new bugs are horrendous :(. I've got it working nicely on Xcode 5.0.2 pre-release, but badly on 5.0.1

For now, I recommend: use Xcode 5.0.2 pre-release, and wait until someone has time to read the new Apple docs.

(please note: Apple screwed-up Xcode 4.5 and 4.6 badly, making it impossible to submit apps to the App Store uf they used libraries. I've seen they've rewritten their docs for 5, but I don't know if they've fixed their 4.6 bugs yet, so I'm afraid of recommending any changes until we're sure / have tested them!)

**Comment by adamgit** [comment]
I've re-written the install instructions, and greatly simplified the front-page.

Everything works fine with latest Xcode5 (apart from a bug that I've given the workaround for - and which I am confident Apple will soon fix anyway)

**Comment by n8v** [comment]
Thank you again for taking the time to write documentation like this. I haven't had a chance to try these updates yet, but as a newbie with XCode and ObjC, clear instructions can save me days of struggle.
